### PR TITLE
Update install_unattended.md

### PR DESCRIPTION
**Pull request by teivanov**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

File: content/en/docs/apim_installation/apiportal_install/install_unattended.md [hunk 1]
```diff
@@ -21,27 +21,40 @@ To install the API Portal software in unattended mode:
...
     * `--initial-ha-instance`: Accepts yY/nN. Flag indicating whether this is the initial instance for HA setup.
     * `--php-ini`            : The directory where `php.ini` file is located. Example: `/etc`.
     * `--apache-config`      : The directory where the Apache configuration files are located. Example: `/etc/httpd/conf.d`.
     * `--use-encryption-key` : Accepts yY/nN. Flag indicating whether to use an encryption key. This option is required for public API mode.
+    * `--encryption-key`     : The place where the encryption key is stored. Example: `/home/encryption/key`. The last segment is the file name (in this example it is called `key`). Used when --use-encryption-key is set to yY.
     * `--use-ssl`            : Accepts yY/nN. Flag indicating whether API Portal will be served by SSL.
     * `--ssl-type`           : Accepts 1 (use custom certificate and private key) or 2 (use self-signed certificate). Indicates what SSL type to use.
-
+    * `--ssl-certificate`    : Path to the SSL certificate. Used when option 1 SSL type is selected.
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_install/install_unattended.md:49]: @teivanov This list is to describe the options used in the example above, not to describe all possible options to use with the script. So, why did you add a couple of properties which aren't used in the example?

I've also noticed a few more properties, that were there for a while, but also not used, so should be removed from this list:

--apache-without-php
--mysql-ssl-method
--ha-instance
--encryption-key

Comment (teivanov) [reply content/en/docs/apim_installation/apiportal_install/install_unattended.md:49]: We want to show the customer all the options that the installation script provides.

Comment (andreamussap) [reply content/en/docs/apim_installation/apiportal_install/install_unattended.md:49]: See - https://github.com/Axway/axway-open-docs/issues/1108

**Review by andreamussap — APPROVED** [review]
Deploy preview https://deploy-preview-1092--axway-open-docs.netlify.app/docs/apim_installation/apiportal_install/install_unattended/

**Comment by andreamussap** [comment]
@teivanov this is done. Thank you for your contribution.

If you're making updates via GitHub, remember to keep your fork up to date - https://axway-open-docs.netlify.app/docs/contribution_guidelines/deleting_a_repository/

**Pull request merged**